heartbeat/dnsupdate.in

```diff
@@ -47,7 +50,7 @@ meta_data() {
 	cat <<END
 <?xml version="1.0"?>
 <!DOCTYPE resource-agent SYSTEM "ra-api-1.dtd">
-<resource-agent name="dnsupdate" version="1.0">
+<resource-agent name="dnsupdate2" version="1.0">
```

Comment:
We cant change the name here, or we'll get issues with pcs and/or pacemaker. You can update the version to e.g. 1.1 instead.